meson.build

```diff
@@ -47,10 +47,10 @@ endif
 lib_deps = [
   cc.find_library('m', required: false),
   dependency('threads'),
-  dependency('libavformat', version: '>= 58.12.100'),
-  dependency('libavfilter', version: '>= 7.16.100'),
-  dependency('libavcodec', version: '>= 58.18.100'),
-  dependency('libavutil', version: '>= 56.14.100'),
+  dependency('libavformat', version: '>= 60.16.100'),
+  dependency('libavfilter', version: '>= 9.12.100'),
+  dependency('libavcodec', version: '>= 60.31.102'),
+  dependency('libavutil', version: '>= 58.29.100'),
```

Comment:
This probably requires a changelog mention

Reply:
Done.